pysilcam/plotting.py

```diff
@@ -51,47 +52,42 @@ def plot(self, imc, imbw, times, d50_ts, vd_mean, display):
         ax.set_ylim(10, 10000)
         ax.set_yscale('log')
 
-        #Plot PSD in lower right axis
-        norm = np.sum(vd_mean['total'].vd_mean)/100
+        # Plot PSD in lower right axis
+        norm = np.sum(vd_mean['total'].vd_mean) / 100
```

Comment:
F841: local variable 'norm' is assigned to but never used  <!-- Automatically posted by Lintly -->